File: WheelsCodingChallenge/Category/UILabel+Ext.m
//
//  UILabel+Ext.m
//  WheelsCodingChallenge
//
//  Created by Mike Saradeth on 4/4/19.
//  Copyright © 2019 Mike Saradeth. All rights reserved.
//

#import "UILabel+Ext.h"

@implementation UILabel (Ext)

- (void)roundedDot {
    self.layer.cornerRadius = self.bounds.size.height / 2;
    self.layer.masksToBounds = true;
}

@end

File: WheelsCodingChallenge/Extensions/UILabel+Ext.swift
//
//  UILabel+Ext.swift
//  WheelsCodingChallenge
//
//  Created by Mike Saradeth on 4/3/19.
//  Copyright © 2019 Mike Saradeth. All rights reserved.
//

import Foundation
import UIKit

extension UILabel {
    func circle() {
        self.layer.cornerRadius = self.bounds.width / 2
        self.layer.masksToBounds = true
    }
}
